mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -5,9 +5,6 @@
 import requests
```

Comment:
The skinny client will eventually need to support `databricks_artifact_repo` for compatibility with Databricks. Leaving this here as a note for myself to remove the dependency on `azure.storage.blob.BlobClient` for this store. We've done this already for a different compatibility purpose on Databricks, so it should be relatively quick to apply this change.